src/objects/index.ts

```diff
@@ -29,7 +28,6 @@ export function start({ stage, props }: StartParameters): void {
   startFonts()
   startStage(stage)
   startConfig(props)
-  startNodesContainer()
```

Comment:
There's no longer a global object for the root nodes container. It can be retrieved from the viewport using `viewport.getChildByName(DEFAULT_NODES_CONTAINER_NAME)`.